ui/tests/acceptance/replication/dr/secondary-test.js

```diff
@@ -8,6 +8,7 @@ module('Acceptance | DR secondary details', function(hooks) {
   setupApplicationTest(hooks);
 
   hooks.beforeEach(function() {
+    this.set('mode', 'dr');
```

Comment:
Makes no difference to test error if I add this or not.  I was trying to see if this would solve the problem.